resources/config/services.xml

```diff
@@ -5,14 +5,14 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="psysh.config" class="Psy\Configuration">
+        <service id="Psy\Configuration">
             <call method="setHistoryFile">
                 <argument>%kernel.cache_dir%/psysh_history</argument>
             </call>
         </service>
 
-        <service id="psysh.shell" class="Psy\Shell" public="true">
-            <argument type="service" id="psysh.config" />
+        <service id="psysh.shell" class="Psy\Shell">
```

Comment:
If I'm not mistaken, like `psysh.shell` is now private, you should go through `self::$container` instead of `self::$kernel->getContainer()` in test file to get it and it should fix travis.

Reply:
👍